### ISSUE TITLE
buttons (inline keyboard) info in message object

### ISSUE THREAD

**Issue by JoeyDieleman78** [python-botogram/botogram#139]
Hi, If a message has inline (keyboard) buttons, it's not available in the message object (when receiving for example with @bot.process_message).
In these buttons there could be spam links, I can't detect them now.

Example of inline keyboards with links attached below:
https://core.telegram.org/bots/2-0-intro

**Comment by MarcoBuster** [comment]
Hi @JoeyDieleman78 and thank you for submitting an issue!
Unfortunately botogram does not support that, but it will be implemented very soon and included in the new release (0.7).

**Comment by JoeyDieleman78** [comment]
Yes I noticed it botofram didn't support it. Great to hear it will be in the next version thanks!

**Comment by MarcoBuster** [comment]
@JoeyDieleman78 keep this issue opened until the feature is merged :)

**Comment by JoeyDieleman78** [comment]
Sorry will keep it open thanks for the fast responses!
